lib/src/base-styles/_nx-colors.scss

```diff
@@ -302,9 +302,10 @@
 
   // NxGlobalSidebar
   --nx-color-global-sidebar-background: var(--nx-swatch-blue-10);
-  --nx-color-global-sidebar-separator-color: var(--nx-swatch-indigo-40);
+  --nx-color-global-sidebar-border: #fff; // note: has no effect in light mode since the border width is 0
+  --nx-color-global-sidebar-separator: var(--nx-swatch-indigo-40);
```

Comment:
There were three variables who's names ended in `-color`, which I felt was redundant since these are all `--nx-color-` variables anyway.  Lmk if you think I should not have done that in this PR